Makefile

```diff
@@ -25,6 +25,7 @@ requirement: roms
 
 roms: $(ROMS)
```

Comment:
There is no rule for 'extern_dependencies' ?

Reply:
https://github.com/Ragarnoy/GBMU/pull/4#discussion_r680335114